id,module,name,file,doc,markers,status,message,duration
functional_correctness_test_folder/post_cut-off/gpt-5/full_docstr/test_snippet_362.py::test_case_0,functional_correctness_test_folder.post_cut-off.gpt-5.full_docstr.test_snippet_362,test_case_0,functional_correctness_test_folder/post_cut-off/gpt-5/full_docstr/test_snippet_362.py,,xfail,failed,[XPASS(strict)] ,0.00023848700220696628
functional_correctness_test_folder/post_cut-off/gpt-5/full_docstr/test_snippet_362.py::test_case_1,functional_correctness_test_folder.post_cut-off.gpt-5.full_docstr.test_snippet_362,test_case_1,functional_correctness_test_folder/post_cut-off/gpt-5/full_docstr/test_snippet_362.py,,,passed,,0.00047454699961235747
functional_correctness_test_folder/post_cut-off/gpt-5/full_docstr/test_snippet_362.py::test_case_2,functional_correctness_test_folder.post_cut-off.gpt-5.full_docstr.test_snippet_362,test_case_2,functional_correctness_test_folder/post_cut-off/gpt-5/full_docstr/test_snippet_362.py,,,failed,Failed: DID NOT RAISE <class 'RuntimeError'>,0.0007239960032165982
